### ISSUE TITLE
Two modes: read/write? 

### ISSUE THREAD

**Issue by MagnusEnger** [PTFS-Europe/koha-plugin-crontab#6]
Would it make sense to be able to run this in two different moeds, one where you can only view the crons, and one where you can edit them too? 

It could be a on/off switch somwhere like koha-conf.xml, or somehow configured differently for different users, so perhaps superlibrarians could edit and others could view? Or even more fine grained, with settings for individual users. 

Related to #2, probably.

**Comment by mrenvoize** [comment]
Definitely, I see that as the main blocker before I more actually build any kpz's for release. I hadn't thought about a simple switch, but that could get this moving a little quicker, I really like the idea of limiting editing with permissions too, it was at the top of the list.

**Comment by lizrea** [comment]
Love the idea of read only mode
